identity-wallet/.gitignore

```diff
@@ -1,16 +1,2 @@
-# Rust
+# .gitignore specific to /identity-wallet
```

Comment:
I think this comment does not really add any further value, but if you prefer to add it to avoid editing the wrong .gitignore file when having many .gitignore files open at the same time, we can leave it in.